tests/features/steps/payment_steps.py

```diff
@@ -216,3 +226,8 @@ def step_impl(context, id):
 def step_impl(context, count):
     assert len(json.loads(context.resp.data)) == int(count)
     assert context.resp.status_code == 200
+
+@then('I should see an error message saying "{error_msg}" with status code "{status_code}"')
+def step_impl(context,error_msg,status_code):
+    assert error_msg in json.loads(context.resp.data)['error']
+    assert context.resp.status_code == int(status_code)
```

Comment:
You can probably use the same convention you used before, with `status.CODE` instead of using a code from the behave string.